wire-library/wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -103,13 +103,35 @@ class SwiftGenerator private constructor(
 
   fun generatedTypeName(type: Type) = type.typeName
 
+  internal val Field.isMessage
+    get() = type!!.isMessage
+  internal val Field.isEnum
+    get() = type!!.isEnum
+  internal val Field.isOptional: Boolean
+    get() = typeName.optional
   internal val Field.isMap: Boolean
     get() = type!!.isMap
   internal val Field.keyType: ProtoType
     get() = type!!.keyType!!
   internal val Field.valueType: ProtoType
     get() = type!!.valueType!!
 
+  private val Field.isCollection: Boolean
+    get() = when (typeName.makeNonOptional()) {
+      DATA, STRING -> true
+      else -> isMap || isRepeated
+    }
+
+  private val Field.codableDefaultValue: String?
+    get() = default?.let {
+      return it
```

Comment:
I'm not sure we'll ever hit this since we shouldn't be calling this for optionals, and optionals are the only fields that can have defaults. Doesn't hurt, but wanted to call it out.